freqtrade/strategy/interface.py

```diff
@@ -268,8 +279,8 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                     sell: bool, low: float = None, high: float = None,
                     force_stoploss: float = 0) -> SellCheckTuple:
         """
-        This function evaluate if on the condition required to trigger a sell has been reached
-        if the threshold is reached and updates the trade record.
+        This function evaluate if one of the conditions required to trigger a sell
```

Comment:
"evaluates"

Reply:
Note to myself: I need to verify if lock_pairs is used in every places where it should be called.

Reply:
the assumption is that it's called whenever a sell happens, which can be 

a) after triggering a sell for whatever reason (including force-sell)
b) after stoploss_on_exchange is filled

I don't think we have another point where sells can be triggered (but checking is better than regretting) :+1: